### ISSUE TITLE
Re-do how output sync works

### ISSUE THREAD

**Issue by ronag** [CasparCG/server#821]

**Comment by premultiply** [comment]
#772

**Comment by TomKaltz** [comment]
Yeah I think I just ran into an issue with this handling. I am using an unsynced extra channel to bring in my decklink producers for routing to other channels. If I route the decklink layer to genlocked channel, the audio becomes garbled after many hours.  Would be nice to be able to configure a channel to derive its sync from another channel.  For now, I think I can work around this issue by brining the decklink inputs in on hidden layers on a genlocked channel so that I can route from there.
